ADL5801 symbol

jcyrax opened the pull request:
Add new symbol for ADL5801 frequency mixer.
[http://www.analog.com/media/en/technical-documentation/data-sheets/ADL5801.pdf](http://www.analog.com/media/en/technical-documentation/data-sheets/ADL5801.pdf)
![symbol](https://user-images.githubusercontent.com/16546947/35488667-b1061c68-0494-11e8-9eb7-32449ebc5be8.png)
![description](https://user-images.githubusercontent.com/16546947/35488669-b2cbd5a6-0494-11e8-983c-c4e9eb1c6c11.png)
![fp_filter](https://user-images.githubusercontent.com/16546947/35488671-b61ab074-0494-11e8-92a9-3d45f690a1b2.png)
![fields](https://user-images.githubusercontent.com/16546947/35488672-b7850a22-0494-11e8-9caa-76efae707108.png)

No aliases.
No footprint in kicad footprint library, but field set accordingly

------------
Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [ ] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required

evanshultz commented:
@jcyrax 
Thanks for contributing to KiCad! Some comments:
1. The datasheet link in your PRs are messed up. For some reason they all point to https://github.com/KiCad/kicad-symbols/pull/url.
1. Please use 100mil pin lengths when pin count is <100. And also for your symbols at https://github.com/KiCad/kicad-symbols/pull/247.
1. Stack the VPLO pins.
1. You can also stack pin 25 since the datasheet says it must be soldered to ground.

I'll review the footprint for this symbol before merging this one.

jcyrax commented:
1. fixed
2. fixed
3. fixed
4. fixed
![image](https://user-images.githubusercontent.com/16546947/35567364-8582159e-05cd-11e8-9e31-1afe375ab82a.png)

evanshultz commented:
@jcyrax Thank you!
